pyproject.toml

```diff
@@ -124,6 +124,9 @@ goes = [
 # Jupyter notebook and lab interface
 jupyter = ["ipywidgets>=7.6", "jupyterlab>=2.2"]
 
+# MetView for working with gridded meteorology forecasts
+metview = ["metview>=1.15"]
```

Comment:
I'm seeing that metview is used in the [specific humidity notebook](https://py.contrails.org/tutorials/interpolating-specific-humidity.html). Previous we weren't using nb-val for either of the `/tutorials/**.ipynb` notebooks. I think we should still skip running nb-val on this particular one, and remove the metview dependency. (The [workflow running the notebook tests isn't able to import metview](https://github.com/contrailcirrus/pycontrails/actions/runs/7543699304/job/20535303263#step:10:1429) because the system libraries aren't found)

Reply:
Revereted adding `metview` in f45e691c1d854ecb4d20399a1e0f485056764053